Jbarget/oxymore
# polishes project icons with responsive styles #134

### What changes have you made?
- repositioned the icons on desktop to look a bit closer to original designs
- extended mobile styles to bigger screen sizes (I think this gives a better UX)
- reduced the view height on the project grid just to centre everything a bit  

### Which issue(s) does this PR fix?
Fixes #134 

### Screenshots (if there are design changes)
<img width="1436" alt="Screenshot 2020-09-15 at 20 15 44" src="https://user-images.githubusercontent.com/53219789/93248542-44369300-f790-11ea-9b40-f9e1b7e6a4cc.png">

### How to test
- check that the icons appear as a random cluster on desktop and in a single column on smaller screen sizes
- make sure that whenever the screen is scollable that the UP button always appears
